conftest.py

```diff
@@ -0,0 +1,5 @@
+from lr_face.models import Architecture
```

Comment:
moet dit bestandje per se in de root? Anders is in de tests map netter?

Reply:
Done